### ISSUE TITLE
[Bug-Candidate]: no back color support for terminal screen.xterm-256color

### ISSUE THREAD

**Issue by rappie** [crytic/echidna#1135]
### Describe the issue:

This happens very rarely at random moments. No way to reproduce.

Might be similar to the earlier "cups" error I kept getting in the past.

### Version:

Latest 2.2.1 `master` built with `nix-env`.

### Relevant log output:

```shell
echidna: no back color support for terminal screen.xterm-256color
CallStack (from HasCallStack):
  error, called at src/Graphics/Vty/Output/TerminfoBased.hs:127:32 in vty-5.38-Dbd4gS2zkOy8uP4PqnkPG0:Graphics.Vty.Output.TerminfoBased
```

**Comment by rappie** [comment]
Possibly related:
```
echidna: user error (Terminal does not define required capability "sgr0")
```

**Comment by ggrieco-tob** [comment]
Is this solved in 2.2.3?

**Comment by rappie** [comment]
I have not had these TERM related errors for a while now since updating to the latest version

**Comment by ggrieco-tob** [comment]
Great, let's close. Feel free to reopen if you see these again.

**Comment by rappie** [comment]
```
echidna: user error (Terminal does not define required capability "clear")
```
Just had this one. Should we reopen it or create a new issue?

**Comment by elopez** [comment]
@rappie if you're able to reproduce it, can you check if running `stty sane` or `tput reset` on the terminal help fix it?

**Comment by rappie** [comment]
I've only had it once so far, but I regularly get a situation where CTRL+C is not working anymore after running Echidna. If I press it, the ^C character appears but it does not stop/cancel anything, it is just a character that I can remove with backspace.

To fix it, I do `reset`. I assume this is the same as `tput reset`?. I will try your commands next time.

**Comment by rappie** [comment]
```
echidna: no back color support for terminal screen.xterm-256color
CallStack (from HasCallStack):
  error, called at src/Graphics/Vty/Output/TerminfoBased.hs:127:32 in vty-5.38-t4uBxudScr4gwaMP3IHWw:Graphics.Vty.Output.TerminfoBased
```

Just had the exact same error as mentioned in the original issue. It is very rare now though.

**Comment by rappie** [comment]
I haven't had any issues with this in a very long time. Shall we close the issue?

**Comment by gustavo-grieco** [comment]
Good idea, I was not able to reproduce this as well, probably some faulty dependency.
